### PR TITLE
Add metrics events for Wyre and CoinSwitch

### LINKED ISSUE #7699: Add metrics event for completed eth purchase
So we can measure how frequently users are using our on-ramps (from our metrics dashboard).

### PR DESCRIPTION
**Pull request by whymarrh**
Closes #7699

This PR adds metrics events for the ETH purchase buttons.

### CODE REVIEW

File: ui/app/components/app/modals/deposit-ether-modal/deposit-ether-modal.component.js [hunk 1]
```diff
@@ -141,7 +142,16 @@ export default class DepositEtherModal extends Component {
               title: this.context.t('buyWithWyre'),
               text: this.context.t('buyWithWyreDescription'),
               buttonLabel: this.context.t('continueToWyre'),
-              onButtonClick: () => toWyre(address),
+              onButtonClick: () => {
+                this.context.metricsEvent({
+                  eventOpts: {
+                    category: 'Onboarding',
```
Comment (Gudahtt) [inline ui/app/components/app/modals/deposit-ether-modal/deposit-ether-modal.component.js:148]: Does the `Onboarding` category make sense here? 🤔

Comment (whymarrh) [reply ui/app/components/app/modals/deposit-ether-modal/deposit-ether-modal.component.js:148]: It's primarily an way to _onboard_ people

Comment (Gudahtt) [reply ui/app/components/app/modals/deposit-ether-modal/deposit-ether-modal.component.js:148]: Not really? It's a way to buy Eth. Existing users need to buy Eth too. It's not even presented during onboarding.

Comment (whymarrh) [reply ui/app/components/app/modals/deposit-ether-modal/deposit-ether-modal.component.js]: Updated to use "Accounts"

Comment (whymarrh) [reply ui/app/components/app/modals/deposit-ether-modal/deposit-ether-modal.component.js]: Let me know if you have a different suggestion

Comment (Gudahtt) [reply ui/app/components/app/modals/deposit-ether-modal/deposit-ether-modal.component.js]: Maybe "On-Ramp"? Or "Integrations", or "Third-Party", or "Exchange"? 🤷‍♂️ 
A new category would just have this one action type in it, but that's not unprecedented. We'll likely have more related metrics soon as well.

I guess "Accounts" is fine too.

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**